README.md

```diff
@@ -46,15 +45,53 @@ The only way to debug is to build and run the main app from IDE:
 6. In logjoint.mac project's Properties, in section Run->Configurations->Default put Arguments `--plugin /path/to/logjoint-symphony-plugin/plugin/bin/Debug/netstandard2.0`
 7. Run
 
+## Integration testing
+
+Plugin integration tests verify that plugin binaries work with the host application. Integration test cases are programmed into a netstandard2.0 library logjoint.symphony.plugin.integration.dll. That library is shipped together with other plugin binaries to allow logjoint backend run the tests whenever it needs to verify the plugin.
+
+To run integration tests
+```
+dotnet logjoint.plugintool test bin/symphony.zip ~/Applications/logjoint.app/Contents/MonoBundle
```

Comment:
I tried above test command and got the following failure:
dotnet logjoint.plugintool test bin/symphony.zip ~/Applications/logjoint.app/Contents/MonoBundle

Unhandled Exception: System.ArgumentException: Can not find required logjoint.integration.tests.dll in location: /Users/bertil.jeppsson/Applications/logjoint.app/Contents/MonoBundle
   at LogJoint.PluginTool.Program.FindHostIntegrationTestsAsm(String startDirectory)
   at LogJoint.PluginTool.Program.Test(String[] args)
   at LogJoint.PluginTool.Program.Main(String[] args)
   at LogJoint.PluginTool.Program.<Main>(String[] args)

Is the integration test DLL shipped with the official application so I need to update, or do I need to specify a local logjoin debug folder @sergey-su ?

Reply:
The logjoint.integration.tests.dll that it complain about is a part of the application now. Please check for updates.

Reply:
It says I'm up-to-date when I scan for updates, but still there is no such DLL. 
Did you publish? I guess I can try to run it against a local build of the main app too, but it seems the DLL have not been published, or my updated doesn't detect the latest version. @sergey-su

Reply:
When I try to run it locally using my custom built logjoing from latest master I get the following error:

dotnet logjoint.plugintool test bin/symphony.zip ~/code/tools/logjoint/trunk/platforms/osx/bin/debug/logjoint.app/Contents/MonoBundle/

Unhandled Exception: System.ArgumentException: Plug-in does not contain tests: /var/folders/xv/g0c95f7x62gdxxgnzsh6cp85qdw1j4/T/logjoint.int.tests.bin.ddcf6f36f816407ba46e42cd2eefa251/manifest.xml
   at LogJoint.Tests.Integration.TestRunner.RunPluginTests(String pluginDir, String filters, Boolean needsDepsDownload) in /Users/bertil.jeppsson/code/tools/logjoint/trunk/tests/integration/Utils/TestRunner.cs:line 60
   at LogJoint.PluginTool.Program.Test(String[] args)
   at LogJoint.PluginTool.Program.Main(String[] args)
   at LogJoint.PluginTool.Program.<Main>(String[] args)

Reply:
It says it is up to date indeed. But I can try to re-install from scratch to be 100%

Reply:
So I downloaded the latest one from GitHub and now I do have the integration test DLL.
find ~/Applications/logjoint.app/Contents/MonoBundle/ -name logjoint.integration.tests.dll
/Users/bertil.jeppsson/Applications/logjoint.app/Contents/MonoBundle//logjoint.integration.tests.dll

So I'm not sure why the updated thought I was on latest even though I wasn't. 

Anyway when I try to run the tests again from this folder:
`logjoint-symphony-plugin/plugin`
Run command:
`dotnet logjoint.plugintool test bin/symphony.zip ~/Applications/logjoint.app/Contents/MonoBundle`

I get the following error:
```
Unhandled Exception: System.ArgumentException: Plug-in does not contain tests: /var/folders/xv/g0c95f7x62gdxxgnzsh6cp85qdw1j4/T/logjoint.int.tests.bin.8281db7525704aa9bab9ce6fe6beafdf/manifest.xml
   at LogJoint.Tests.Integration.TestRunner.RunPluginTests(String pluginDir, String filters, Boolean needsDepsDownload)
   at LogJoint.PluginTool.Program.Test(String[] args)
   at LogJoint.PluginTool.Program.Main(String[] args)
   at LogJoint.PluginTool.Program.<Main>(String[] args)
```

I built it from the root of the repo using:
`dotnet build plugin`

If I try to run the test command from the root folder as below I get this error:
```
dotnet logjoint.plugintool test plugin/bin/symphony.zip ~/Applications/logjoint.app/Contents/MonoBundle
Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET Core program, but dotnet-logjoint.plugintool does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
```

Reply:
I guess you did not not do packaging step, so the bin/symphony.zip was old. In fact that command `dotnet logjoint.plugintool test bin/symphony.zip ...` was explained for education purposes. Actual testing job should normally be done by `pack.sh` that was also updated in this PR. 

Anyway, pushed another commit that makes the testing example self-sufficient.